now-build.sh

```diff
@@ -2,5 +2,4 @@
 
 yarn
 yarn build
-yarn build:props
```

Comment:
👍 
Is this temporary while we aren't using it?

Reply:
Yup, whilst we’re not using the artifacts this adds like 5 minutes to the build. As we don’t get concurrent website builds, it just seems to be slowing us right now